locales/en/dashboard.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Let's move translations into common.js. I think we don't want to group the translations into different files.